docs/painless/painless-contexts/painless-runtime-fields-context.asciidoc

```diff
@@ -9,29 +9,32 @@ about how to use runtime fields.
 *Methods*
 
 --
-`emit` (Required)::
+// tag::runtime-field-emit[]
+`emit`:: (Required)
         Accepts the values from the script valuation. Scripts can call the
         `emit` method multiple times to emit multiple values.
 +
-IMPORTANT: The `emit` method cannot accept `null` values. Do not call this method if
-the referenced fields do not have any values.
+The `emit` method applies only to scripts used in a 
+<<painless-execute-runtime-context,runtime fields context>>.
++
+IMPORTANT: The `emit` method cannot accept `null` values. Do not call this
+method if the referenced fields do not have any values.
 +
 .Signatures of `emit`
 [%collapsible%open]
 ====
 The signature for `emit` depends on the `type` of the field.
 
 [horizontal]
-[[emit-boolean]]   `boolean`::   `emit(boolean)`
-[[emit-date]]      `date`::      `emit(long)`
-[[emit-double]]    `double`::    `emit(double)`
-[[emit-geo-point]] `geo_point`:: `emit(double lat, double long)`
-[[emit-ip]]        `ip`::        `emit(String)`
-[[emit-long]]      `long`::      `emit(long)`
-[[emit-keyword]]   `keyword`::   `emit(String)`
-
+`boolean`::   `emit(boolean)`
```

Comment:
I think the anchors were useful. They don't change the output but they let us link folks directly to the signature when they have a question. I know I'd find it useful on slack and discuss.

Reply:
Agreed -- I'll add them back. I removed them when I was fussing with that table and forgot to re-add.